Unable to install ffmpeg from source

## Description
ffmpeg fails to build when installing from source.

If you executed a specific command:
```
chronos@localhost ~ $ crew install -s ffmpeg
```

Output:
```
...

License: nonfree and unredistributable
Creating configuration files ...
awk: run time error: redirection not allowed in sandbox mode
        FILENAME="" FNR=0 NR=0
awk: run time error: redirection not allowed in sandbox mode
        FILENAME="" FNR=0 NR=0
awk: run time error: redirection not allowed in sandbox mode
        FILENAME="" FNR=0 NR=0
awk: run time error: redirection not allowed in sandbox mode
        FILENAME="" FNR=0 NR=0
Preconfiguring package...
make: *** No rule to make target 'install-libs-yes', needed by 'install-libs'.  Stop.
ffmpeg failed to install: `make DESTDIR=/usr/local/tmp/crew/dest install` exited with 2
chronos@localhost ~ $ 
```

## Versions and system information
chromebrew: `0.4.3`

ChromeOS: `chronos@localhost ~ $ cat /etc/lsb-release 
CHROMEOS_ARC_ANDROID_SDK_VERSION=25
CHROMEOS_ARC_VERSION=4510202
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_RELEASE_BOARD=eve-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=22
CHROMEOS_RELEASE_BUILDER_PATH=eve-release/R64-10176.22.0
CHROMEOS_RELEASE_BUILD_NUMBER=10176
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=64
CHROMEOS_RELEASE_DESCRIPTION=10176.22.0 (Official Build) beta-channel eve 
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=beta-channel
CHROMEOS_RELEASE_VERSION=10176.22.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=10176.22.0`

Model: Pixelbook

Architecture: `x86_64`

--- a/packages/ffmpeg.rb
+++ b/packages/ffmpeg.rb
@@ -3,21 +3,21 @@ require 'package'
 class Ffmpeg < Package
   description 'A complete, cross-platform solution to record, convert and stream audio and video.'
   homepage 'https://ffmpeg.org/'
-  version '3.3.3-1'
-  source_url 'https://ffmpeg.org/releases/ffmpeg-3.3.3.tar.xz'
-  source_sha256 'd2a9002cdc6b533b59728827186c044ad02ba64841f1b7cd6c21779875453a1e'
+  version '3.4.1'
+  source_url 'https://ffmpeg.org/releases/ffmpeg-3.4.1.tar.xz'
+  source_sha256 '5a77278a63741efa74e26bf197b9bb09ac6381b9757391b922407210f0f991c0'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ffmpeg-3.3.3-1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ffmpeg-3.3.3-1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/ffmpeg-3.3.3-1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ffmpeg-3.3.3-1-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ffmpeg-3.4.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ffmpeg-3.4.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/ffmpeg-3.4.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ffmpeg-3.4.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '4748d900703d05d05fdb584b534ffd6ea1b5ecad3a78187804d23af3254dd87e',
-     armv7l: '4748d900703d05d05fdb584b534ffd6ea1b5ecad3a78187804d23af3254dd87e',
-       i686: 'a85f9c79aaefe5b80bf59c2067c6a9b16b5c96edc604c49d33c3d5ac9ffb3a0d',
-     x86_64: '8e78f2b9f7bd47a7f9f04e157bf441a8a7c54d5c32833b20d38ac9d10300e43e',
+    aarch64: '69b5a4c0f2e48c31a33cbd1f99df02d662d483e14d24a0bcd50da8312fc6cd82',
+     armv7l: '69b5a4c0f2e48c31a33cbd1f99df02d662d483e14d24a0bcd50da8312fc6cd82',
+       i686: '4315c7f32fe05c13cbd1d97950dddfe259c84432406b6f83cd7c548b512788bb',
+     x86_64: 'cf4569ffb65ab474528d3cbf68189b4e49de323d15bb19360ba6c5ebc2bf245f',
   })
 
   depends_on 'gnutls'
@@ -42,6 +42,8 @@ class Ffmpeg < Package
 
   def self.build
     system "TMPDIR=#{CREW_BREW_DIR} ./configure \
+             --prefix=#{CREW_PREFIX} \
+             --libdir=#{CREW_LIB_PREFIX} \
              --arch=#{ARCH} \
              --enable-gpl \
              --enable-nonfree \

